addons/account/views/account_move_views.xml

```diff
@@ -692,14 +692,21 @@
                                 string="Send &amp; Print"
                                 invisible="state != 'posted' or not is_being_sent and not invoice_pdf_report_id or move_type not in ('out_invoice', 'out_refund', 'in_invoice', 'in_refund')"
                                 data-hotkey="y"/>
-                        <!-- Register Payment (only invoices / receipts) -->
+                        <!-- Register Payment (only invoices / receipts, no outstanding payments) -->
                         <button name="action_register_payment" id="account_invoice_payment_btn"
                                 type="object" class="oe_highlight"
                                 invisible="state != 'posted' or payment_state not in ('not_paid', 'partial') or move_type not in ('out_invoice', 'out_refund', 'in_invoice', 'in_refund', 'out_receipt', 'in_receipt')"
                                 context="{'dont_redirect_to_payments': True, 'display_account_trust': True}"
                                 string="Register Payment" data-hotkey="g"
                                 groups="account.group_account_invoice"/>
-                        <!-- Preview (only customer invoices) -->
+                         <!-- Register Payment (only invoices / receipts, with outstanding payments) -->
+                        <button name="action_register_payment"
```

Comment:
find it a bit sad to add a new button in the view for just a change in the invisible or am i missing something :thinking: also the comment is not well aligned

Reply:
I couldn't make it work with t-attf-class, so in the end left it like this

Reply:
Yeah my bad apparently conditional class only work with templates and not views